### PR TITLE
updated hab (0.9.0-20160816000307) Add Docker dependency

### PR DESCRIPTION
**Pull request by davejlong**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/hab.rb [hunk 1]
```diff
@@ -8,5 +8,7 @@
   homepage 'www.habitat.sh'
   license :apache
 
+  depends_on formula: 'docker'
```
Comment (jawshooah) [inline Casks/hab.rb]: The homepage says that it specifically requires Docker for Mac, so this should be `depends_on cask: 'docker'`

**Comment by davejlong** [comment]
Habitat really just requires the Docker CLI. Docker for Mac is the easiest way to get up and running, but it also would work with Docker Machine.

**Comment by jawshooah** [comment]
I see. Let's keep the Cask dependency since that's what they explicitly recommend, and users can always skip that dependency with `--skip-cask-deps`.

**Comment by jawshooah** [comment]
Thanks!

**Pull request merged**